metrotranscom/doorway
# chore: remove shared services

# Pull Request Template

## Issue Overview


## Description

With the partners site updated, there are now no packages that depend on `shared-services`.  This PR removes `shared-services`, updates the Dockerfile, and removes all comments, references, etc.

## How Can This Be Tested/Reviewed?

Verify that the public and partner sites build as expected by running `yarn install`, `yarn build`, and `yarn test:unit` in each directory.

You can verify that the Dockerfile works by either building via `docker build` (passing in all required build args), or by running this command to build via the buildspec file.

`codebuild_build.sh -i public.ecr.aws/codebuild/amazonlinux2-x86_64-standard:5.0 -a cb_out -b ci/buildspec/build_public.yml -e sites/public/.env -c`

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
